docs/slides/content/priority-sliders.md

```diff
@@ -112,10 +112,12 @@ How did you reach consensus?
 
 
 #### Facilitation Tips
-* Topics are not product features but product area are team-building topics
-* Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritisation
-* The Customer team is the one defining priorities
-* it's not static - when the work we're doing takes us towards one priority, it may be that things can change. As such, treat this as a dynamic artefact
+* Topics are not product features but product area are team-building topics.
+<!--Tech Editor: I don't understand this first bullet, and "product area are team building topics" has a subject-verb agreement error and is unclear. The first part is ok, "topics are not product features", but the second part confused me. Please edit for better clarity. -->
```

Comment:
just asked for clarification. I'll update here later on :)

Reply:
A quick scan but I think it was meant to say `Topics are not product features but product area or team-building topics`

Reply:
```suggestion
```

Thanks @springdo :) @nmuller66 let's update that sentence as Donal shared and remove the comment - then we can merge it :)